_data/sites.json

```
@@ -2006,6 +2006,16 @@
         ]
     },
 
+    {
+        "name": "ebonus",
+        "url": "https://ebonus.gg/contact",
+        "difficulty": "hard",
+        "notes": "You must request account deletion at https://ebonus.gg/contact. They will ask for confirmation using email, It takes 30 days for the account to be deleted.",
```

Since the URL is already this link, that can be removed.
```suggestion
        "notes": "After submitting a contact form, they will ask for confirmation using email. After confirmation, account deletion takes 30 days.",
```